raiden/log_config.py

```diff
@@ -160,6 +160,17 @@ def tracebackexception_format(self, *, chain=True):
     TracebackException.format = tracebackexception_format
 
 
+def configure_logging_defaults(logger_level_config: Dict[str, str] = None):
+    if logger_level_config:
+        logger_level_config = dict(logger_level_config)
+    else:
+        logger_level_config = dict()
+
+    logger_level_config.setdefault('filelock', 'ERROR')
+    logger_level_config.setdefault('', DEFAULT_LOG_LEVEL)
```

Comment:
Won't this overwrite my settings when I supply `:DEBUG` to the command line?

Reply:
nope:

```python
>>> d = {}
>>> d.setdefault('a', 1)
1
>>> d
{'a': 1}
>>> d.setdefault('a', 2)
1
>>> d
{'a': 1}
```